src/chi-vue/src/components/data-table/DataTable.tsx

```diff
@@ -303,6 +304,36 @@ export default class DataTable extends Vue {
     this.$emit(DATA_TABLE_EVENTS.SELECTED_ROWS_CHANGE, selectedRowsData);
   }
 
+  _checkIndeterminate(rowData: DataTableRow) {
+    let parent = rowData.parent;
+
+    while (parent) {
+      const parentCheck = document.querySelector(
+        `#checkbox-select-${this._rowId(parent.rowNumber)}`
+      ) as HTMLInputElement;
+      const children = parent.nestedContent.table.data;
```

Comment:
I guess not all rows may have children:
```suggestion
      const children = parent.nestedContent?.table.data;
```